code/modules/modular_computers/file_system/programs/command/comm.dm

```diff
@@ -112,6 +113,28 @@
 		return program.can_run(user, program.computer.get_network())
 	return 1
 
+/datum/nano_module/program/comm/proc/get_shunt()
+	if(isnull(program?.computer))
+		return FALSE
+
+	var/obj/comp = program.computer.get_physical_host()
+
+	if(isnull(comp))
+		return FALSE
+
+	var/obj/effect/overmap/visitable/ship/sector = map_sectors["[comp.z]"]
+
+	if(!sector)
+		return FALSE
+
+	for(var/obj/machinery/ftl_shunt/core/C in SSmachines.machinery)
```

Comment:
should probably be tracked by overmap object, steal how shieldgens do it

Reply:
Probably worth generalizing the method the drills use to a general proc and sharing it around.